### DISCUSSION TITLE
Support for subdomains

### DISCUSSION THREAD

**Discussion by b0Stark** [diced/zipline#424] [Ideas]
Would be nice if zipline could support subdomains for serving content.
I.e.,
Images: `https://{{image}}.domain.tld/xxxx`
Links: `https://{{short}}.domain.tld/xxxx` | Or even a secondary domain: `https://ex.tld/xxxx`
API: `https://{{api}}.domain.tld`

Quite frankly, I wouldn't mind mixing the content either so any kind of resource would show up as:
`https://domain.tld/xxxx`

**Comment by diced** [comment]
Adding this would be pretty difficult with how Zipline works currently, and wouldn't really make sense as there isn't much of a use case for this.
